### DISCUSSION TITLE
[3차 스프린트 - 백엔드] 08-07 숙제 검사(문서화, 로그백, API 생성, 테스트, 로키)

### DISCUSSION THREAD

**Discussion by Minjoo522** [woowacourse-teams/2024-devel-up#251] [백로그 논의]
#### 논의가 필요한 내용

- [x] 로빈 : 로깅 및 모니터링 도식화
- [x] 구름 : 로그백 조사
- [x] 릴리 : 미션 제출 API 기능 구현
- [x] 아톰 : 솔루션 리스트 테스트, 솔루션 상세 테스트, swagger, 미션 스타트
- [x] 리브 : 로키 조사, 로깅 프레임워크, 모니터링 선정 이유 문서화

숙제 관련 자료 코멘트 달아주세요!

**Comment by robinjoon** [comment]
![모니터링 및 로깅 아키텍처](https://github.com/user-attachments/assets/619e266e-0f29-431a-a61d-ba8a13e3c618)

[모니터링 및 로깅 문서](https://github.com/woowacourse-teams/2024-devel-up/wiki/%EB%B0%B1%EC%97%94%EB%93%9C-%EC%84%9C%EB%B2%84-%EB%AA%A8%EB%8B%88%ED%84%B0%EB%A7%81-%EB%B0%8F-%EB%A1%9C%EA%B9%85)

**Comment by Minjoo522** [comment]
[🍀 로깅 프레임워크, 모니터링 선정 이유](https://robinjoon.notion.site/689c0da118d04e93aad353e8453aad76?pvs=4)
[🧡 로키](https://minjoo522.notion.site/86b3f6120f9c418596a959f932c3c96d?pvs=4)

**Comment by alstn113** [comment]
[logback](https://alstn113.notion.site/Logback-2024-08-08-a2c33673a1f2428a9f4f9b52c777951c)
